Automattic/jetpack
# VideoPress: video poster control - re-organize - typescriptify

On this PR:
* Re-write the poster image control with TypeScript
* Create a simple VideoPosterCard component
* Minor implementation changes

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: video poster control - janitorial
  * Re-write the poster image control with TypeScript
  * Create a simple VideoPosterCard component
  * Minor implementation changes

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






_No changes in terms of functionality_

* Go to block editor
* Add/edit a video block
* Change the video poster image
* Confirm it works as expected:
  * Add a poster image
  * Replace by another one
  * Remove it
